boards/weact/stm32g474_core/doc/index.rst

```diff
@@ -0,0 +1,91 @@
+.. zephyr:board:: weact_stm32g474_core
+
+The WeAct STM32G474 Core Board is based on the STM32G474CBU6 microcontroller. The board provides essential features in a compact form factor with USB Type-C connectivity.
+
+Hardware Features
+================
+
+USB Type-C Configuration
+-----------------------
+
+The board features a USB Type-C interface with configurable routing through solder bridges:
+
++---------------+---------+-----------------------------------------------+
+| Solder bridge | Default | Description                                   |
++===============+=========+===============================================+
+| SB3/SB4       | Open    | Connect PA9/PA10 to CC routing               |
++---------------+---------+-----------------------------------------------+
+| SB5/SB6       | Open    | Connect PB6/PB4 to CC routing                |
++---------------+---------+-----------------------------------------------+
+| SB7/SB8       | Closed  | Connect CC1/CC2 to pull-down resistors       |
++---------------+---------+-----------------------------------------------+
+
+Supported Hardware Features
+--------------------------
+
```

Comment:
Add a line similar to https://github.com/zephyrproject-rtos/zephyr/commit/887f210153b8a50e417232582762167d914c4033#diff-96c74ad50824463df0bd48ff926ecd261e62399a80a09226f40891629f2ea148R25